schema/schema.json

```
@@ -0,0 +1,115 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/catppuccin/cli/raw/main/schema/schema.json",
+  "title": "Catppuccin config",
+  "description": "JSON schema for Catppuccin port configuration file",
+  "type": "object",
+  "additionalProperties": false,
```

I'd appreciate if the top schema checked for required values, but allowed additional properties.
IMO the CLI should only lint the `installer` section, would like @ghostx31 & @DakshG07 input on this.

---

```suggestion
  "additionalProperties": true,
```
I recall this conversation happening a while ago, so I will change this to `true` preemptively.  
wrt required fields, let me know if I should remove `app_name` and `binary_name` from the schema as well.